optionals/compat_rhs_afrf3/nodoors/CfgVehicles.hpp

```diff
@@ -0,0 +1,112 @@
+class CfgVehicles {
+    class MRAP_02_base_F;
+    class Offroad_01_base_F;
+    class Truck_F;
+    class O_Truck_02_covered_F;
+    class Heli_Attack_02_base_F;
+    class RHS_Mi8_base;
+    class RHS_Mi8mt_vvs;
+    class RHS_Mi8MTV3_vvs;
+    class Tank_F;
+    class rhs_bmp1_vdv;
+    class rhs_bmp2_vdv;
+    class Wheeled_APC_F;
+    class APC_Tracked_02_base_F;
```

Comment:
I'd rather have the base classes sorted before their child classes.

Reply:
it's generated config and sorting your way is not really necessary. I you insist I can implement it, but I think it's not worth it. The same is in [nouniformrestrictions](https://github.com/acemod/ACE3/blob/master/optionals/nouniformrestrictions/CfgVehicles.hpp) component.